PerfectDay/Repositories/IRepositoryGoal.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.EntityFrameworkCore;
+using PerfectDay.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PerfectDay.Repositories
+{
+    public class IRepositoryGoal<TEntity> : IRepository<TEntity> where TEntity : class
```

Comment:
Please add getAll goals functionality for the repository.

Reply:
Done